scripts/run.py

```diff
@@ -55,18 +57,24 @@ def main(args):
 
   try:
     if options.js_env:
-      s_file = os.path.join(temp_dir, basename + '.s')
-      wast_file = os.path.join(temp_dir, basename + '.wast')
+      if extension == '.c':
+        s_file = os.path.join(temp_dir, basename + '.s')
+        wast_file = os.path.join(temp_dir, basename + '.wast')
+        subprocess.check_call([WACC, '-fno-builtin', options.in_file, '-o', s_file, '-I', '.'])
+        subprocess.check_call([LINKER, s_file, '-o', wast_file])
+      elif extension == '.wast':
+        wast_file = options.in_file
+      else:
+        raise NotImplementedError('unimplemented input file type: %s' % options.in_file)
```

Comment:
NotImplementedError is usually used for unimplemented functions in an abstract class. I'd just use Exception here.